<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrudMadrasahGabungan</class>
 <widget class="QMainWindow" name="CrudMadrasahGabungan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manajemen Data Madrasah</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="filterLayout">
      <item>
       <widget class="QLabel" name="labelJenjang">
        <property name="text">
         <string>Jenjang</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboJenjang"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="cariLayout">
      <item>
       <widget class="QLabel" name="labelCari">
        <property name="text">
         <string>Cari</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputCari"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableMadrasah"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnTambah">
        <property name="text">
         <string>Tambah</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHapus">
        <property name="text">
         <string>Hapus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrint">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
